### DISCUSSION TITLE
Have not get any notification about my github campus expert application

### DISCUSSION THREAD

**Discussion by Harikrishna-HG** [community/community#138948] [GitHub Education]
### Hi everyone,
It's already been two weeks.
![d](https://github.com/user-attachments/assets/12beccfb-31a2-4124-84fd-ef26093093f5)

**Comment by vraj00222** [comment]
there is no update from them yet i am still waiting

**Reply by Harikrishna-HG** [reply]
Thank you for sharing your thought
